subworkflows/local/bam_impute_stitch/tests/main.nf.test

```diff
@@ -15,18 +15,22 @@ nextflow_workflow {
     tag "bcftools"
     tag "bcftools_index"
 
-    test("Impute with stitch two individuals with bamlist") {
```

Comment:
I would keep a similar test as we had before, without renaming, where we pass an empty []:

`test("Impute with stitch two individuals with bamlist and no renaming")`

Reply:
Added